components/schedule/ScheduleBlock.vue

```diff
@@ -66,22 +66,43 @@ export default {
 }
 
 .scheduleBlock.-primary.-active {
-    color: #000000;
-    background-color: #d1ac23;
+    color: #c7c7c7;
+    background-color: #354970;
+    @media (hover: hover) {
+        &:hover {
+            color: #000000;
+            background-color: #d1ac23;
+        }
+    }
```

Comment:
just curious. why can't we get rid of media query here?

```suggestion
    &:hover {
        color: #000000;
        background-color: #d1ac23;
    }
```

Reply:
Without hover media query, mobile users could unintentionally active hover state.